.vscodeignore

```diff
@@ -1,16 +1,14 @@
-# Run 'vsce ls' to verify that the following lines exclude everything not needed
```

Comment:
I suggest we retain the comment line hint about how to check what this file allows into the VSIX